addons/website/views/website_templates.xml

```diff
@@ -1102,5 +1102,16 @@ Sitemap: <t t-esc="url_root"/>sitemap.xml
     </t>
 </template>
 
+<template id="search" name="search">
```

Comment:
Do not use such standard name as "search". Please. If someday you have to grep the code, what will happen do you think ?